src/app/pages/project-overview-page/project-overview-page.component.html

```diff
@@ -53,7 +53,7 @@ <h1 class="mat-h1">{{T.PP.TITLE|translate}}</h1>
           <!--            <mat-icon>flight</mat-icon>-->
           <!--          </button>-->
           <button (click)="remove(project.id)"
-                  [disabled]="project.id === (workContextService.activeWorkContextId$|async)"
```

Comment:
That's not right. The current project should not be delectable, as this might lead to errors. The default project should be delectable however.

Reply:
Oh. Then I guess this issue: https://github.com/johannesjo/super-productivity/issues/581 is incorrect?

I was referring to deleting the current project because otherwise it takes more clicks if you want to delete it. If that's more of an intended thing then this PR and the corresponding issue should be closed.

Reply:
No #581 is correct, but to solve this, one would have to make sure that it is safe to delete a currently active project, which could be done, by making it inactive (e.g. by switching to the today list) before deleting it.